### PR TITLE
Add test for partial update API

### PR DESCRIPTION
**Pull request by rtretyak**

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    @Tidy
+    @Unroll
+    def "Able to partially update flow #data.field without reinstalling its rules"() {
+        given: "A flow"
+        def swPair = topologyHelper.switchPairs.first()
+        def flow = flowHelperV2.randomFlow(swPair)
+        flowHelperV2.addFlow(flow)
+        def originalCookies = northbound.getSwitchRules(swPair.src.dpId).flowEntries
+                                        .findAll { !Cookie.isDefaultRule(it.cookie) }*.cookie
```
[the remark this answers is not in the document]
Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy]: nice, will do

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+                ],
+                [
+                        field   : "priority",
+                        newValue: 654
+                ]
+        ]
+    }
+
+    @Tidy
+    def "Can do partial update on a single-switch flow"() {
```
[the remark this answers is not in the document]
Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy]: I'll rename, but that does not really bring any informational improvement. Being laconic is more about giving good explanation of your test/test steps

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        then: "Flow is left intact"
+        expect northbound.getFlow(flow.flowId), sameBeanAs(flowBeforeUpdate)
+                .ignoring("lastUpdated")
+
+        and: "Flow rules have not been reinstalled"
+        northbound.getSwitchRules(swPair.src.dpId).flowEntries*.cookie.containsAll(originalCookies)
+
+        cleanup: "Remove the flow"
+        flowHelperV2.deleteFlow(flow.flowId)
+    }
```
[the remark this answers is not in the document]
Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy]: not yet implemented

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+}
```
[the remark this answers is not in the document]
Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy]: Which validations do you expect on these fields? I don't think that we have any

[the remark this answers is not in the document]
Comment (andriidovhan) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy]: agree, we don't have any validation for these fields(

**Pull request merged**